src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00acdc;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+.wrapper {
+  margin: 0 50px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.navigation {
+  display: block;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-transform: uppercase;
+  background-color: white;
+  font-size: 12px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+}
+
+.nav_link {
+  text-decoration: none;
+  height: 60px;
+  line-height: 60px;
+  display: block;
+  color: black;
+  justify-content: center;
+  align-self: center;
+  letter-spacing: normal;
+  font-weight: 100;
+}
+
+.nav_item:nth-child(n + 2) {
+  margin-left: 20px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). In this case, you are using `margin-left` which is horizontal margin. It's better to use `margin-right` on every `.nav_item` and remove it from the last child. This way, you won't have extra spacing on the right side of the navigation.